app/components/common/rdsCalendar/index.tsx

```diff
@@ -39,7 +41,10 @@ const RdsCalendar = ({
       defaultDate={defaultDate}
       defaultView={view ?? 'week'}
       style={{ height: height ?? '100vh' }}
-      onSelectEvent={(event) => setCalendarEvent((e) => ({ ...e, event, show: true, new: false }))}
+      onSelectEvent={(event: CalEvent) => {
+        setCalendarEvent((e) => ({ ...e, event, show: true, new: false }));
```

Comment:
Do we need to spread event here? Also what do show and new key refer here. May be we can name it more specifically

Reply:
removed it now